Python/Python.sublime-syntax

```diff
@@ -3,8 +3,9 @@
 name: Python
 file_extensions:
   - py
-  - rpy
   - pyw
+  - pyi
+  - rpy
```

Comment:
I haven't seen `py3` anywhere and it seems fairly non-standard. Do you have any sources on it?